### PR TITLE
[DEVR-113] Global Accounts API Integration

### PR DESCRIPTION
**Pull request by Asfiroth**
- Enable create turnkey accounts
- Enable create zerodev smart contracts

### CODE REVIEW

File: src/middleware.ts [hunk 1]
```diff
@@ -71,16 +87,12 @@ export const middleware = async (
...
-        return NextResponse.redirect(new URL('/app', request.url), {
-          status: 307,
-        });
-      }
-
       if (!isLoginPage) {
         return authMiddleware(request, event);
       }
 
+      console.info('got here');
```
Comment (YokySantiago) [inline src/middleware.ts:94]: Let's remove this console log

File: src/services/globalAccount.ts [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+import {KERNEL_V3_1} from '@zerodev/sdk/constants';
+import { createKernelAccount, createKernelAccountClient, createZeroDevPaymasterClient } from '@zerodev/sdk';
+import { createPublicClient, createWalletClient, http } from 'viem';
+import { createAccount } from '@turnkey/viem';
+import { TStamper } from '@turnkey/http/dist/base';
+import { TurnkeyClient } from "@turnkey/http";
+import { turnkeyApiClient, turnkeyConfig } from '@/config/turnkey';
+
+// configurations
+const PROJECT_ID = 'c3526d90-4977-44e3-8584-8820693a7fd9';
```
Comment (YokySantiago) [inline src/services/globalAccount.ts:20]: Let's add this one to the env variables

**Review by YokySantiago — CHANGES_REQUESTED** [review]
I left some small comments but in general LGTM. Is this deployed in the staging environment?

**Comment by Asfiroth** [comment]
> I left some small comments but in general LGTM. Is this deployed in the staging environment?

Hi @YokySantiago addressed the comments, and wanted to ask you about staging: why are staging deployments disabled?

**Pull request merged**